### ISSUE TITLE
[Publications] Design Review of Movement partnership landscape analysis

### ISSUE THREAD

**Issue by sabrinang** [mozilla/foundation.mozilla.org#6118]
related to issue #6074

Complete a design review once Insights team turn the Movement partnership landscape analysis doc research into a Publication.

### Deadline: 
From Kasia:
~~• We need this help for when we’ll be moving the content online, **Feb 18 - Feb 26**.~~
• aiming to freeze text on by March 24 and publication planned for March 30.

**Comment by taisdesouzalessa** [comment]
@sabrinang added deadlien to this ticket

**Comment by kristinashu** [comment]
Do they have an updated timeline for this?

**Comment by jamilasnell** [comment]
"The movement analysis will take a longer while till we are clear to move content online (at least 2-3 weeks) so you can unblock Sabrina on it." Probably week of the 15th, they will move content online

**Comment by sabrinang** [comment]
Timeline update from Kasia vis slack:
> Publication planned for March 30, we hope to have the text frozen by March 24.  


I'll conduct the design review end of next week then when text has froze and after @taisdesouzalessa gets images in it.

**Comment by sabrinang** [comment]
Laura and team slacked this morning that the specific part they're working on and that should be launched on Wednesday is here: https://foundation.mozilla.org/cms/pages/1994/edit/

Kasia mentioned everything is ready for review https://foundation.mozilla.org/en/data-futures-lab/data-for-empowerment/ (other sections have been live already) but that the Ecosystem study and Glossary (both in draft mode) are especially urgent.

I'll be reviewing it shortly.

**Comment by sabrinang** [comment]
Reviews given last week on Ecosystem Landscape Analysis Review and Data for Empowerment series. Laura/Solana applied changes. Overall the team used the template really well and it went live last week https://foundation.mozilla.org/en/data-futures-lab/data-for-empowerment/

Kasia gave some feedback on the callout box component where they wanted to use it in a few ways but I let her know it doesn't have rich text styling so it limited what they could do so I suggested to just use the content component instead. I can file an issue where it can be callout can be improved so it would be utilized. I let them know we had advanced tables but we can document it on [how do i wagtail](https://foundation.mozilla.org/en/docs/how-do-i-wagtail/publications/).

**Created related follow up issues:**
[Publications] Update How-To Page with Updates (advanced table block etc.) https://github.com/MozillaFoundation/Design/issues/815
[Publications] Callout Box Styling https://github.com/mozilla/foundation.mozilla.org/issues/6509
